src/composables/sdk13.ts

```diff
@@ -119,7 +119,9 @@ export function useSdk13({ store }: IDefaultComposableOptions) {
       },
     });
 
-    connectFrames(sdk);
+    if (IN_FRAME && !FramesConnection.initialized) {
+      FramesConnection.init(sdk);
```

Comment:
This is not required. I have deleted this in one of my PRs https://github.com/aeternity/superhero-wallet/pull/1940/files#diff-c600d713ab89213f35d7d831633578c909c46b00436f1428d8cffb3986f462be

@CedrikNikita please confirm. in /lib/wallet.js file there is already code that does manage iframe connection management. This looks duplicate.